CRM/Contribute/Form/Task/PDFLetter.php

```diff
@@ -167,4 +167,14 @@ public function postProcess() {
     CRM_Contribute_Form_Task_PDFLetterCommon::postProcess($this);
   }
 
+  /**
+   * List available tokens for this form.
+   *
+   * @return array
+   */
+  public function listTokens() {
+    $tokens = CRM_Core_SelectValues::contactTokens();
```

Comment:
This should have also merged in contributionTokens. Fixing this in https://issues.civicrm.org/jira/browse/CRM-17580